UM/Qt/qml/UM/RecolorImage.qml

```diff
@@ -20,6 +21,7 @@ Item
         visible: false
         sourceSize.width: parent.width
         sourceSize.height: parent.height
+        layer.enabled: true
```

Comment:
I'll change this to `layer.enabled: mirror` to save some rendering performance. These images are used a lot (e.g. with every setting) so performance is important.